src/pages/studyView/chartHeader/ChartHeader.tsx

```diff
@@ -79,8 +82,11 @@ export interface ChartControls {
     isShowNAChecked?: boolean;
     showNAToggle?: boolean;
     showSwapAxes?: boolean;
+    showResultsPageButton?: boolean;
     showSurvivalPlotLeftTruncationToggle?: boolean;
     survivalPlotLeftTruncationChecked?: boolean;
+    showXAxisAtTop?: boolean;
+    xAxisAtTop?: boolean;
```

Comment:
do we really need this? are we planning to show mutation plot x-axis on top in the study view page?

Reply:
Currently, the Mutation Plot in the Results View Page cannot be made larger or smaller. But this is available in StudyView. Now when the chart is made bigger, the lollipops have much greater height. I personally was feeling harder to read the values on the x-axis with higher lollipops and thought of adding this. I am not sure if those values matter much in analyzing the data. So let me know if this needs to be removed. @onursumer